src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -803,6 +804,7 @@ private static String processPageSourceForGodMode(String content) {
                 .replace(TestProperties.inst().TEST_UNREG_ACCOUNT, "${test.unreg}")
                 .replace(Config.SUPPORT_EMAIL, "${support.email}")
                 // today's date
+                .replace(TimeHelper.formatDate(now).replace("/", "&#x2f;"), "{*}")
```

Comment:
I get this failure, probably related to this line of code?
![image](https://cloud.githubusercontent.com/assets/3910365/9760612/21a89318-5729-11e5-868e-46de70768cd3.png)